feature-libs/tracking/schematics/add-tracking/schema.json

```diff
@@ -13,8 +13,26 @@
     },
     "lazy": {
       "type": "boolean",
-      "description": "Lazy load the personalization features.",
+      "description": "Lazy load the tracking features.",
       "default": true
+    },
+    "features": {
+      "type": "array",
+      "uniqueItems": true,
+      "items": {
+        "enum": [
+          "Personalization",
+          "Google Tag Manager",
```

Comment:
What do you think about prefixing them with tag management?

eg. `Tag management / Google Tag Manager`, so we see that those 2 are part of "tag managers" family?

Now it looks like any of those features are top level feature, but in reality it is personalization / tag management

Reply:
Done